TombEngine/Game/collision/collide_room.cpp

```diff
@@ -943,11 +823,11 @@ short GetNearestLedgeAngle(ItemInfo* item, CollisionInfo* coll, float& distance)
 
 			// Get front floor block
 			auto room = GetRoomVector(item->Location, Vector3i(ffpX, y, ffpZ)).RoomNumber;
-			auto block = GetCollision(ffpX, y, ffpZ, room).Block;
+			auto sectorPtr = &GetPointCollision(Vector3i(ffpX, y, ffpZ), room).GetSector();
```

Comment:
I am against using `Ptr` suffix everywhere. We can already tell the data type by just peeking definition or seeing whether `.` or `->` is applicable in every case.